tests/SprykerTest/Zed/Econda/Business/Exporter/Writer/File/NameGenerator/CsvNameGeneratorTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerTest\Zed\Econda\Business\Exporter\Writer\File\NameGenerator;
```

Comment:
Namespace for **ALL** tests in `"spryker-eco/*"` should be `SprykerEcoTest`!!!

**this applies to every test class below**

Reply:
Done